ReSharper.FSharp/src/FSharp/FSharp.Psi.Services/src/Daemon/Highlightings/FSharpErrorUtil.fs

```diff
@@ -11,6 +12,12 @@ open JetBrains.ReSharper.Plugins.FSharp.Util
 open JetBrains.ReSharper.Psi.ExtensionsAPI
 open JetBrains.ReSharper.Psi.Tree
 
+type DiagnosticTypeInfo =
```

Comment:
Let's call it something like `FSharpDiagnosticTypeInfo`, so it's clearer it's coming from the plugin?